apps/omg_watcher_rpc/test/omg_watcher_rpc/web/controllers/transaction_test.exs

```diff
@@ -699,18 +718,25 @@ defmodule OMG.WatcherRPC.Web.Controller.TransactionTest do
                    "payments" => [
                      %{"amount" => alice_to_bob, "currency" => @eth_hex, "owner" => bob_addr}
                    ],
-                   "fee" => %{"amount" => fee, "currency" => @eth_hex},
+                   "fee" => %{"currency" => @default_fee_currency},
                    "metadata" => metadata
                  }
                )
 
       assert Utxo.position(blknum, txindex, oindex) |> Utxo.Position.encode() == utxo_pos
     end
 
-    @tag fixtures: [:alice, :bob, :more_utxos]
-    test "returns correctly formed transaction, identical with the verbose form", %{alice: alice, bob: bob} do
+    @tag fixtures: [:alice, :bob, :more_utxos, :test_server]
+    test "returns correctly formed transaction, identical with the verbose form", %{
+      alice: alice,
+      bob: bob,
+      test_server: context
+    } do
       alias OMG.State.Transaction
 
+      response = TestServer.make_response(@fee_response)
```

Comment:
first see the comments about the `fee_response` being "customized" for tests. If the customizations can be undone, then you could move this to `setup`.

It seems that the fee response is auxiliary to `transaction.create` tests, so it could be uniformly done in `setup` for all tests.

Unless you wanted to test some interactions between those two behaviors :thinking:.

Reply:
I tried to keep the same test data that we had before to make it faster so I didn't have to understand each test individually. I could simplify some, but others still need a custom fee response.
Some of these tests will most likely be updated soon anyway with the work that @T-Dnzt is doing (preventing fee overpay).